src/server/common/helpers/file/file-utils.js

```diff
@@ -1,18 +1,25 @@
 import { compressPdf } from './pdf-compression.js'
 import { compress as compressImage } from './image-compression.js'
 import { fileSizeInMB } from './size.js'
+import { GetObjectCommand } from '@aws-sdk/client-s3'
+import { config } from '~/src/config/config.js'
 
 /**
  * Handles the uploaded file by processing it based on its content type.
  * Compresses the file if it is a PDF or an image, logs the compression details,
  * and returns the processed file.
  * @param {object} req - The request object containing the S3 method to retrieve the file.
- * @param {string} filePath - The path to the file in the S3 bucket.
+ * @param {object} uploadedFile - The path to the file in the S3 bucket.
```

Comment:
Does the description here need updating?

Reply:
Also, It would be ideal if we can be more specific here & in logger than `{object}` - is that something you've tried & run into some issues with?